client/package-lock.json

```diff
@@ -2492,7 +2492,7 @@
     },
     "bl": {
       "version": "1.2.2",
-      "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
+      "resolved": "http://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
```

Comment:
Something weird is happening with the lockfile. You may want to run `npm install --prefer-online` on root.